core/bentley/src/StatusCategory.ts

```diff
@@ -133,8 +137,10 @@ class UnknownError extends HTTP.InternalServerError { }
 
 class NotImplemented extends HTTP.NotImplemented { }
 
-function lookupCategory(error: BentleyError): StatusCategory {
-  switch (error.errorNumber) {
+class Aborted extends HTTP.BadRequest { }
```

Comment:
Most accurate HTTP status code for `Aborted` errors would be 204, but it would require a breaking change on `IModelReadRpcInterface`, so for now we treat `Aborted` as HTTP 400 "Bad Request".

GitHub issue will be open once this PR is merged to switch to 204 for iTwin.js 5.0.

Reply:
https://github.com/iTwin/itwinjs-core/issues/6475